pkgs/development/libraries/geoclue/2.0.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = libintlOrEmpty ++
    [ intltool glib json_glib libsoup geoip
-     dbus dbus_glib avahi
+     dbus dbus_glib avahi wrapGAppsHook
```

Comment:
`wrapGAppsHook` should be moved to `nativeBuildInputs`. And while you are at it, can you also move `intltool` there?

Reply:
Fixed, thanks!